draft-ietf-scitt-architecture.md

```diff
@@ -192,6 +192,12 @@ Auditor:
 
 : an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service.
 
+Client:
+
+: an application making protected Transparency Service resource requests on behalf of the resource owner and with its authorization {{RFC6749}}.
```

Comment:
Are you intending to make all SCITT clients OAUTH clients?

Seems like you just want to borrow the definition

Reply:
The current suggested text is likely to trigger a "should be a normative reference" comment.

Reply:
agreed with Orie